DataConnectors/Darktrace.json

```diff
@@ -0,0 +1,114 @@
+{
+    "id": "DarktraceDarktrace",
+    "title": "Darktrace",
+    "publisher": "Darktrace",
+    "descriptionMarkdown": "Common Event Format (CEF) is an industry standard format on top of Syslog messages, used by many security vendors to allow event interoperability among different platforms. By connecting your Darktrace config to Azure Sentinel, you can take advantage of search & correlation, alerting, and threat intelligence enrichment for each log.",
```

Comment:
Please update the description to something more specific for Darktrace, in terms of log types being pulled in, etc.